bin/propolis-server/src/lib/initializer.rs

```diff
@@ -864,9 +865,10 @@ impl<'a> MachineInitializer<'a> {
                 .unwrap(),
             bios_rom_size: ((rom_size / (64 * 1024)) - 1) as u8,
             // Characteristics-not-supported
```

Comment:
style nit: Since you're turning these into nice named values, the accompanying comments can probably be tossed out now (here and below)